resources/views/organizations/show-role-selection.blade.php

```diff
@@ -4,13 +4,12 @@
         <h1>
             {{ __('Create new community organization') }}
         </h1>
-
-        <h2>{{ __('Create new community organization') }}</h2>
     </x-slot>
 
     <form class="stack" action="{{ localized_route('organizations.save-roles', $organization) }}" method="post" novalidate>
         <fieldset class="field @error('type') field--error @enderror">
-            <legend>{{ __('What type of organization are you?') }}</legend>
+            <legend>{{ __('Your organization’s role') }}</legend>
+            <x-hearth-hint for="roles">{{ __('Please tell us what your organization would like to do here. You must pick at least one of these roles. You can always change this later.') }} <a target="_blank" href="{{ localized_route('about.for-community-organizations') }}">Learn more about these roles.</a> </x-hearth-hint>
```

Comment:
"Learn more about these roles" is not properly wrapped in `__('')` and thus won't be picked up for translation. Please make sure it is processed like line 11 above.